docs/manage/configuration.md

```diff
@@ -41,18 +42,51 @@ Edit the file directly or use `asdf local` (or `asdf global`) which updates it.
 
 ## `$HOME/.asdfrc`
 
-Add a `.asdfrc` file to your home directory and asdf will use the settings specified in the file. The file should be formatted like this:
+Add an `.asdfrc` file to your home directory and asdf will use the settings specified in the file. The file below shows the required format with the default values to demonstrate:
 
 ```:no-line-numbers
-legacy_version_file = yes
+legacy_version_file = no
+use_release_candidates = no
+always_keep_download = no
+plugin_repository_last_check_duration = 60
 ```
 
-**Settings**
+### `legacy_version_file`
+
+Plugins with support can read the versions files used by other version managers, for example, `.ruby-version` in the case of Ruby's `rbenv`.
+
+| Options                                                    | Description                                                   |
+| :--------------------------------------------------------- | :------------------------------------------------------------ |
+| `no` <Badge type="tip" text="default" vertical="middle" /> | Use `.tool-versions` to read versions                         |
+| `yes`                                                      | Use plugin fallback to legacy version files (`.ruby-version`) |
+
+### `use_release_candidates`
+
+Configure the `asdf update` command to upgrade to the latest Release Candidate instead of the latest Semantic Version.
+
+| Options                                                    | Description               |
+| :--------------------------------------------------------- | :------------------------ |
+| `no` <Badge type="tip" text="default" vertical="middle" /> | Semantic Version is used  |
+| `yes`                                                      | Release Candidate is used |
+
+### `always_keep_download`
+
+Configure the `asdf install` command to keep or delete the source code or binary it downloads.
+
+| Options                                                    | Description                                |
+| :--------------------------------------------------------- | :----------------------------------------- |
+| `no` <Badge type="tip" text="default" vertical="middle" /> | Delete source code or binary after install |
+| `yes`                                                      | Keep source code or binary after install   |
```

Comment:
Tables seem a little unnecessary for some of these options.

Also, this:

> Delete source code or binary after install

Should be:

> Delete source code or binary after successful install

Reply:
>Tables seem a little unnecessary for some of these options.

I found the the `plugin_repository_last_check_duration` very wordy and unclear without the table. After adding one it seemed best to be consistent, though I agree the binary settings don't benefit all that much.